### PR TITLE
[Charts] sync services

### PR DESCRIPTION
**Pull request by jbygdell**
* adds missing config to `sync`
* adds missing parts to `syncapi`
* removes unused options
* Fixes a typo in the MQ chart

### CODE REVIEW

**Review by blankdots — COMMENTED** [review]
can we make these configurable as well in `syncapi`:
- `SYNC_API_ACCESSIONROUTING`
- `SYNC_API_INGESTROUTING`
- `SYNC_API_MAPPINGROUTING`

**Review by blankdots — DISMISSED** [review]

**Review by blankdots — APPROVED** [review]

File: charts/sda-svc/README.md [hunk 1]
```diff
@@ -135,13 +135,26 @@ Parameter | Description | Default
...
 `global.inbox.s3Region` | S3 inbox region. |`""`
 `global.inbox.s3ChunkSize` | S3 chunk size in MB. |`15`
 `global.inbox.s3AccessKey` | Access key to S3 inbox . |`null`
 `global.inbox.s3SecretKey` | Secret key to S3 inbox. |`null`
 `global.inbox.s3CaFile` | CA certificate to use if the S3 inbox is internal. |`null`
 `global.inbox.s3ReadyPath` | Endpoint to verify that the inbox is respondig. |`""`
+`global.sync.api.password` | Password for authenticating to the syncAPI server | `null`
+`global.sync.api.user` | User for authenticating to the syncAPI server | `null`
+`global.sync.centerPrefix` | Prefix for locally generated datasets | `null`
+`global.sync.destination.storageType` | Storage type for the sync destination, currently only supports S3 | `s3`
```
Comment (pontus) [inline charts/sda-svc/README.md:149]: Why do we default other `storageType`s  to `null` but this to `s3`?

File: charts/sda-svc/README.md [hunk 2]
```diff
@@ -172,6 +181,12 @@ Parameter | Description | Default
 `credentials.ingest.mqPassword` | Broker password for ingest | `""`
 `credentials.intercept.mqUser` | Broker user for intercept  | `""`
 `credentials.intercept.mqPassword` | Broker password for intercept | `""`
+`credentials.sync.dbUser` | Databse user for sync | `""`
```
Comment (pontus) [inline charts/sda-svc/README.md:184]:
```suggestion
`credentials.sync.dbUser` | Database user for sync | `""`
```

Rypo.

File: charts/sda-svc/templates/intercept-certificate.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+{{- if eq "federated".Values.global.schemaType }}
```
Comment (pontus) [inline charts/sda-svc/templates/intercept-certificate.yaml:1]:
```suggestion
{{- if eq "federated" .Values.global.schemaType }}
```

File: charts/sda-svc/templates/intercept-deploy.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.intercept.deploy}}
+{{- if eq "federated".Values.global.schemaType }}
```
Comment (pontus) [inline charts/sda-svc/templates/intercept-deploy.yaml:1]:
```suggestion
{{- if eq "federated" .Values.global.schemaType }}
```

File: charts/sda-svc/templates/intercept-secrets.yaml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.intercept.deploy}}
+{{- if eq "federated".Values.global.schemaType }}
```
Comment (pontus) [inline charts/sda-svc/templates/intercept-secrets.yaml:1]:
```suggestion
{{- if eq "federated" .Values.global.schemaType }}
```

**Review by pontus — DISMISSED** [review]

File: charts/sda-svc/README.md [hunk 6]
```diff
@@ -135,13 +135,26 @@ Parameter | Description | Default
...
+`global.sync.api.user` | User for authenticating to the syncAPI server | `null`
+`global.sync.centerPrefix` | Prefix for locally generated datasets | `null`
+`global.sync.destination.storageType` | Storage type for the sync destination, currently only supports S3 | `s3`
+`global.sync.destination.s3Accesskey` | Access key to S3 sync destination | `null`
+`global.sync.destination.s3Bucket` | sync destination bucket | `null`
+`global.sync.destination.s3Port` | Port that the S3 sync destination instance is available on | `443`
+`global.sync.destination.s3Secretkey` | Secret key to S3 sync destination | `null`
+`global.sync.destination.s3url` | URL to S3 sync destination instance. | `null`
+`global.sync.remote.host` | URL to jthe remote syncAPI host | `null`
+`global.sync.remote.password` | Password for connectiong to the remote syncAPI host | `null`
```
Comment (aaperis) [inline charts/sda-svc/README.md:156]:
```suggestion
`global.sync.remote.password` | Password for connecting to the remote syncAPI host | `null`
```

File: charts/sda-svc/README.md (hunk 2 above)
Comment (aaperis) [reply charts/sda-svc/README.md:184]: fyi, the same typo is in many places

File: charts/sda-svc/values.yaml [hunk 7]
```diff
@@ -235,17 +234,17 @@ global:
       configFile: "iss.json"
       iss:
         - iss: "https://login.elixir-czech.org/oidc"
-          jku: "https://login.elixir-czech.org/oidc//oidc/jwks"
+          jku: "https://login.elixir-czech.org/oidc/jwks"
```
Comment (aaperis) [inline charts/sda-svc/values.yaml:237]:
```suggestion
          jku: "https://login.elixir-czech.org/oidc/jwk"
```

File: charts/sda-svc/README.md (hunk 1 above)
Comment (jbygdell) [reply charts/sda-svc/README.md:149]: Because the current implementation only works with S3

**Review by blankdots — APPROVED** [review]

**Review by blankdots — APPROVED** [review]

**Review by aaperis — APPROVED** [review]

**Pull request merged**